lib/src/base-styles/_nx-colors.scss

```diff
@@ -26,6 +26,24 @@
   --nx-color-interactive-border-focus: var(--nx-swatch-teal-35);
   --nx-color-interactive-shadow-focus: hsla(var(--nx-swatch-teal-hs), 35%, 0.5);
 
+  --nx-color-primary-button-background: var(--nx-swatch-blue-40);
+  --nx-color-primary-button-background-hover: var(--nx-swatch-blue-30); // also focus
+  --nx-color-primary-button-background-active: var(--nx-swatch-blue-50);
+  --nx-color-primary-button-text: var(--nx-swatch-white);
+  --nx-color-secondary-button: var(--nx-swatch-blue-40); //border and text
+  --nx-color-secondary-button-background: var(--nx-swatch-white);
+  --nx-color-secondary-button-background-hover: var(--nx-swatch-blue-90);
+  --nx-color-secondary-button-background-active: var(--nx-swatch-blue-95);
+  --nx-color-tertiary-button-border: var(--nx-swatch-grey-50);
+  --nx-color-tertiary-button-text: var(--nx-color-text-stark);
+  --nx-color-tertiary-button-background-focus: transparent;
+  --nx-color-tertiary-button-background-active: var(--nx-swatch-indigo-97);
+  --nx-color-error-button-background: var(--nx-color-alert-error);
```

Comment:
In Sketch, the background of the error button is 40 in light mode, but 35 in dark mode

Reply:
> In Sketch, the background of the error button is 40 in light mode, but 35 in dark mode

I wasn't totally sure how/if to relate that to the `--nx-color-alert-error` variable in dark mode like it is in light mode. 
 Dark mode alerts are not yet implemented, but looking at their mockups it seems like they probably won't use the `--nx-color-alert-...` variables as all of the icons are the same dark color.  So for the `--nx-color-error-button-background` dark mode override, I just set it to the swatch color directly.  f93f61b83

Reply:
> So for the `--nx-color-error-button-background` dark mode override, I just set it to the swatch color directly. 

Thanks for clarifying, makes sense!